packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.test.tsx

```diff
@@ -0,0 +1,216 @@
+        })),
+        getHeaderProps: () => ({ key: 'header-key' }),
+        getTableProps: () => ({}),
+        getRowProps: () => ({ key: 'row-key' }),
+      };
+      return children(mockTableProps);
+    }
+    return children;
+  };
+
+  return {
+    DataTableSkeleton: jest.fn(({ id = 'default' }) => <div data-testid={`data-table-skeleton-${id}`}>Loading...</div>),
+    Button: jest.fn(({ children, onClick, kind, renderIcon, hasIconOnly, iconDescription }) => (
+      <button onClick={onClick} data-kind={kind} title={iconDescription}>
+        {hasIconOnly ? 'icon' : children}
+      </button>
+    )),
+    Search: jest.fn(({ placeholder, onChange, value }) => (
+      <input type="text" placeholder={placeholder} onChange={onChange} value={value} data-testid="search-input" />
+    )),
+    Form: jest.fn(({ children, onSubmit }) => (
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+          onSubmit?.(e);
+        }}
+      >
+        {children}
+      </form>
+    )),
+    DataTable: DataTableMock,
+    Table: jest.fn(({ children }) => <table>{children}</table>),
+    TableContainer: jest.fn(({ children }) => <div>{children}</div>),
+    TableHead: jest.fn(({ children }) => <thead>{children}</thead>),
+    TableRow: jest.fn(({ children, className }) => <tr className={className}>{children}</tr>),
+    TableHeader: jest.fn(({ children }) => <th>{children}</th>),
+    TableBody: jest.fn(({ children }) => <tbody>{children}</tbody>),
+    TableCell: jest.fn(({ children }) => <td>{children}</td>),
+    Layer: jest.fn(({ children, className }) => <div className={className}>{children}</div>),
+  };
+});
+
+const mockPanelData = {
+  panels: [
+    {
+      id: '1',
+      conceptUuid: 'test-uuid-1',
+      name: 'Complete Blood Count',
+      effectiveDateTime: '2024-01-01T10:00:00Z',
+      value: '120',
+      relatedObs: [
+        {
+          id: '1-1',
+          name: 'Hemoglobin',
+          value: '14',
+          meta: { units: 'g/dL', range: '12-16' },
+        },
+      ],
+    },
+    {
+      id: '2',
+      conceptUuid: 'test-uuid-2',
+      name: 'Liver Function Test',
+      effectiveDateTime: '2024-01-02T10:00:00Z',
+      value: '35',
+      relatedObs: [
+        {
+          id: '2-1',
+          name: 'ALT',
+          value: '35',
+          meta: { units: 'U/L', range: '7-56' },
+        },
+      ],
+    },
+  ],
+  isLoading: false,
+  groupedObservations: {
+    'test-uuid-1': [
+      {
+        id: '1',
+        conceptUuid: 'test-uuid-1',
+        name: 'Complete Blood Count',
+        value: '120',
+        effectiveDateTime: '2024-01-01T10:00:00Z',
+      },
+    ],
+    'test-uuid-2': [
+      {
+        id: '2',
+        conceptUuid: 'test-uuid-2',
+        name: 'Liver Function Test',
+        value: '35',
+        effectiveDateTime: '2024-01-02T10:00:00Z',
+      },
+    ],
+  },
+};
+
+describe('PanelView', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
This gets taken care of [automatically](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/jest.config.js#L6).